projects/vdk-plugins/vdk-jupyter/deploy-tutorial.md

```diff
@@ -0,0 +1,51 @@
+Learn how to prepare and deploy a Data Job using VDK in Jupyter Notebook UI.
+Time Commitment: About 5-10 minutes.
+
+## Prerequisites
+ * Installed VDK, Control Service, Jupyter instance and Jupyter extension
+ * Created and developed a VDK Data Job
+
+## Deploying a job
+
+This page will assume you have already created a job both locally and in the cloud,
+and have developed the job.
+
+
+
+0. Ensure your Data Job is in a production-ready state. This means tagging all production cells as VDK cells, and untagging any
+   you do not want to be ran during the Data Job execution through the side menu shown in the image.
+
+![Example of how to tag cells in Jupyter](./celltagging.png)
+
+1. Navigate to the data job directory.
+
+![VDK dropdown menu](./vdk-menu.png)
```

Comment:
Since new modifications ave been done on the menu please use the toolbar pictures from these dialogs: https://miro.com/app/board/uXjVMwyYjto=/?share_link_id=860687647726